<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametryzationProfilesWdgtTopper</class>
 <widget class="QWidget" name="ParametryzationProfilesWdgtTopper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Working directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Consumer address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="lblWorkingDirectory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="leConsumerAddress">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QPushButton" name="tbNIchecker">
        <property name="text">
         <string>Check</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
          <normaloff>:/katynko/svg4/sc_expandpage.svg</normaloff>:/katynko/svg4/sc_expandpage.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QPushButton" name="tbUpdateCompleter">
        <property name="text">
         <string>Update</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
          <normaloff>:/katynko/svg/view-refresh.svg</normaloff>:/katynko/svg/view-refresh.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>NI Checker</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="lblWarnings">
        <property name="text">
         <string/>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QPushButton" name="tbSetWorkingDir">
        <property name="text">
         <string>Change</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
          <normaloff>:/katynko/svg/document-open.svg</normaloff>:/katynko/svg/document-open.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../Matilda-units/gui/image-and-font/res.qrc"/>
 </resources>
 <connections/>
</ui>
